frontend/src/components/common/menu.jsx

```diff
@@ -8,17 +8,16 @@ export default class Menu extends React.Component {
         super(props);
         this.toggle = this.toggle.bind(this);
         this.close = this.close.bind(this);
-        this.handleToggleClick = this.handleToggleClick.bind(this);
-        this.handleMenuClick = this.handleMenuClick.bind(this);
+        this.onClickOutside = this.onClickOutside.bind(this);
         this.state = { show: false };
     }
 
     componentDidMount() {
-        document.addEventListener('click', this.close, false);
+        document.addEventListener('click', this.onClickOutside, false);
```

Comment:
This change was so that the menu for the filters stays open even after you click a toggle